test/core/event_engine/test_suite/posix_event_engine_test.cc

```diff
@@ -19,18 +19,26 @@
 #include <grpc/grpc.h>
 
 #include "src/core/lib/event_engine/posix_engine/posix_engine.h"
+#include "src/core/lib/experiments/experiments.h"
 #include "test/core/event_engine/test_suite/event_engine_test.h"
+#include "test/core/event_engine/test_suite/oracle_event_engine_posix.h"
 #include "test/core/util/test_config.h"
 
 int main(int argc, char** argv) {
   testing::InitGoogleTest(&argc, argv);
   grpc::testing::TestEnvironment env(&argc, argv);
+  if (!grpc_core::IsPosixEventEngineEnablePollingEnabled()) {
+    return 0;
```

Comment:
This will skip all timer tests as well if EventEngine polling is not enabled for the gRPC binary. I don't think we want that, the tests should be exercised regardless.

Reply:
Same answer as before.